android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/ProjectsParser.kt

```diff
@@ -165,6 +165,8 @@ class ProjectsParser : BaseParser() {
                     mProject.noCUDAPref = mCurrentElement.toString() != "0"
                 } else if (localName.equals(Project.Fields.NO_ATI_PREF, ignoreCase = true)) {
                     mProject.noATIPref = mCurrentElement.toString() != "0"
+                } else if (localName.equals(Project.Fields.DISK_USAGE, ignoreCase = true)) {
+                    mProject.diskUsage = mCurrentElement.toDouble()
```

Comment:
Please update unit test to include this new line

Reply:
Can you have a link to upper lines that have a unit test?

Reply:
Should be this: https://github.com/BOINC/boinc/blob/master/android/BOINC/app/src/test/java/edu/berkeley/boinc/rpc/ProjectInfoParserTest.kt